Src/Offline-Resources.psm1

```diff
@@ -54,6 +54,7 @@ $OptimizeOffline.DevMode = (Resolve-FullPath -Path $OptimizeOffline.Packages -Ch
 $OptimizeOffline.WindowsStore = (Resolve-FullPath -Path $OptimizeOffline.Packages -Child WindowsStore)
 $OptimizeOffline.MicrosoftEdge = (Resolve-FullPath -Path $OptimizeOffline.Packages -Child MicrosoftEdge)
 $OptimizeOffline.Win32Calc = (Resolve-FullPath -Path $OptimizeOffline.Packages -Child Win32Calc)
+$OptimizeOffline.DormantDefender = (Resolve-FullPath -Path $OptimizeOffline.Packages -Child DormantDefender)
```

Comment:
Can you please explain me what shall the Packages/DormantDefender folder server for?